### PR TITLE
improvement and implementation

### PR DESCRIPTION
**Pull request by DotRacel**
automatically remove annotation after obfuscation and implemented loader adder.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,8 +15,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>10</source>
-                    <target>10</target>
+                    <source>11</source>
+                    <target>11</target>
```
Comment (superblaubeere27) [inline pom.xml]: Is there a reason for bumping the Java version?

File: pom.xml [hunk 2]
```diff
@@ -36,8 +36,8 @@
 
     <repositories>
         <repository>
-            <id>ObjectWeb</id>
-            <url>https://repository.ow2.org/nexus/content/repositories/releases/</url>
+            <id>sonatype-nexus-snapshots</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (superblaubeere27) [inline pom.xml]: What is the problem with the `ObjectWeb` repository?

File: pom.xml [hunk 3]
```diff
@@ -52,6 +52,11 @@
             <artifactId>asm-tree</artifactId>
             <version>${asm.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-commons</artifactId>
+            <version>${asm.version}</version>
+        </dependency>
```
Comment (superblaubeere27) [inline pom.xml]: Is this dependency needed?

**Review by superblaubeere27 — CHANGES_REQUESTED** [review]
Thank you for implementing this feature. I am fine with everything but the `pom.xml` changes.

File: pom.xml (hunk 3 above)
Comment (DotRacel) [reply pom.xml]: sorry, they're not supposed to be changed.
i just reverted them back.

**Comment by DotRacel** [comment]
> Thank you for implementing this feature. I am fine with everything but the `pom.xml` changes.

sorry, they're not supposed to be changed.
i just reverted them back.

**Review by superblaubeere27 — APPROVED** [review]

**Comment by superblaubeere27** [comment]
Thank you for your contribution!

**Pull request merged**